core/key.go

```diff
@@ -47,7 +47,6 @@ const (
 	COLLECTION_NAME                = "/collection/name"
 	COLLECTION_SCHEMA_VERSION      = "/collection/version"
 	COLLECTION_INDEX               = "/collection/index"
-	SCHEMA_MIGRATION               = "/schema/migration"
 	SCHEMA_VERSION                 = "/schema/version/v"
 	SCHEMA_VERSION_HISTORY         = "/schema/version/h"
```

Comment:
Good spot, thanks :)

- [ ] Redo SCHEMA_VERSION_HISTORY

> question: Why is the history of change no longer something we want to track?

It is tracked, via collections.  Otherwise it has no use and would be dead code to do so.